DotnetCLISdkVersion.txt

```diff
@@ -0,0 +1 @@
+2.0.0-preview1-005899
```

Comment:
There actually isn't a line ending on this file. There isn't in the other one either.